client/dist/style.css

```diff
@@ -27,4 +27,12 @@ container {
   height: 1000px;
   width: 680px; 
   border: 10px solid green;
+}
```

Comment:
The instructions on the roadmap specified to use either Styled Components or CSS Modules and gave us this link. Try to convert them. I used the styled components, you'll need to add a dependency to use that. Let me know if you need help with that. https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822